### DISCUSSION TITLE
GDPR web hooks URLs

### DISCUSSION THREAD

**Discussion by stephenkeable** [Shopify/shopify-app-template-node#1002] [Q&A]
Wanted to check with the addition of the GDPR web hooks in CLI 3.x :-
https://github.com/Shopify/shopify-app-template-node/blob/cli_three/web/gdpr.js
Which is great help, thanks.

Would the URLs for the Mandatory GDPR webhooks in the App Setup screen of the partner dashboard all three be:-
https://myapp.com/api/webhooks

Or have I misunderstood?

**Comment by mingfengwan** [comment]
I tested this and you are correct.
